HACKING.md

```diff
@@ -15,7 +15,11 @@ Create and activate a new virtual environment:
 
 Make sure pip is up-to-date:
 
-    (snapcraft) $ pip install --upgrade pip
+   (snapcraft) $ pip install --upgrade pip
+    
+Change directory to Snapcraft:
+
+   (snapcraft) $ cd snapcraft
```

Comment:
oh, but we are never mentioning here about the clone.
What about mentioning to fork the github repo, and give the command to clone your fork?